docs/source/frameworks/pytorch.rst

```diff
@@ -2,55 +2,158 @@
+    :caption: `train.py`
 
-    import pandas as pd
     import torch
-    import bentoml
+    import torch.nn as nn
+    import torch.nn.functional as F
+    import torch.optim as optim
+
+
+    class Net(nn.Module):
+        def __init__(self):
+            super().__init__()
+            self.conv1 = nn.Conv2d(3, 6, 5)
+            self.pool = nn.MaxPool2d(2, 2)
+            self.conv2 = nn.Conv2d(6, 16, 5)
+            self.fc1 = nn.Linear(16 * 5 * 5, 120)
+            self.fc2 = nn.Linear(120, 84)
+            self.fc3 = nn.Linear(84, 10)
 
-    class NGramLanguageModeler(nn.Module):
+        def forward(self, x):
+            x = self.pool(F.relu(self.conv1(x)))
+            x = self.pool(F.relu(self.conv2(x)))
+            x = x.view(-1, 16 * 5 * 5)
+            x = F.relu(self.fc1(x))
+            x = F.relu(self.fc2(x))
+            x = self.fc3(x)
+            return x
 
-    def __init__(self, vocab_size, embedding_dim, context_size):
-        super(NGramLanguageModeler, self).__init__()
-        self.embeddings = nn.Embedding(vocab_size, embedding_dim)
-        self.linear1 = nn.Linear(context_size * embedding_dim, 128)
-        self.linear2 = nn.Linear(128, vocab_size)
 
-    def forward(self, inputs):
-        embeds = self.embeddings(inputs).view((1, -1))
-        out = F.relu(self.linear1(embeds))
-        out = self.linear2(out)
-        log_probs = F.log_softmax(out, dim=1)
-        return log_probs
+    model = Net()
+    ... # training
 
-    model = NGramLanguageModeler(len(vocab), EMBEDDING_DIM, CONTEXT_SIZE)
+    # =========
 
-    # save the model to model store:
-    tag = bentoml.pytorch.save_model("ngrams", model)
+    bentoml.pytorch.save(model, "my_torch_model")
 
-    # get a BentoModel (a reference to model in model store) by tag:
-    metadata = bentoml.models.get(tag)
 
-    # load the model back in memory:
-    model = bentoml.pytorch.load_model("ngrams:latest")
+``bentoml.pytorch`` also supports saving models that take multiple tensors as input:
+
+.. code-block:: python
+    :caption: `train.py`
+
+    import torch
+    import torch.nn as nn
+    import torch.nn.functional as F
+    import torch.optim as optim
 
-    # Run a given model under `Runner` abstraction with `to_runner`
-    input_data = pd.from_csv("/path/to/csv")
-    runner = bentoml.pytorch.get(tag).to_runner()
-    runner.init_local()
-    runner.run(pd.DataFrame("/path/to/csv"))
+
+    class Net(nn.Module):
+        def __init__(self):
+            super().__init__()
+
+        def forward(self, x, y):
+            return x + y
+
+
+    model = Net()
+    ... # training
+
+    bentoml.pytorch.save(model, "my_torch_model")
 
 .. note::
 
-   You can find more examples for **PyTorch** in our `gallery <https://github.com/bentoml/gallery>`_ repo.
+    :bdg-info:`Remarks:` All of the external python modules/util functions that the model
+    requires must be included in the project. The PyTorch framework requires them to restore
+    the model.
```

Comment:
makes this a warning. 

Also add an example how people should add the modules / utility functions to bento via `bentofile.yaml`.

You can use the YOLO example here.